spring-cql/src/main/java/org/springframework/cassandra/config/CassandraCqlClusterFactoryBean.java

```diff
@@ -366,6 +372,16 @@ public void setAuthProvider(AuthProvider authProvider) {
 	}
 
 	/**
+	 * Set the {@link NettyOptions}.
```

Comment:
For?

Perhaps something a little more descriptive, like...

"Set the {@link NettyOptions} used by a client to customize the driver's underlying Netty layer."